benchmarks/bench_trtllm_gen_fused_moe_autotuner.py

```
@@ -1,20 +1,112 @@
+FLOAT4_E2M1_MAX = 6.0
 
-def bench_trtllm_gen_fused_moe_autotuner(
+def fp8_quantize(x):
+    max = x.float().abs().nan_to_num().max()
+    scale = FLOAT8_E4M3_MAX / max
+    x = (x * scale).to(torch.float8_e4m3fn)
+    return x, 1.0 / scale
+
+def bench_trtllm_gen_fused_moe_autotuner_fp8(
+    tune_max_num_tokens: Optional[int],
+    quant_mode: Literal["Fp8-Per-Tensor"],
```

Ruff (0.14.3)

Unused function argument: `quant_mode`

(ARG001)